submitted_models/ctu_cras_norlab_spot_sensor_config_2/model.config

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<model>
+  <name>ctu_cras_norlab_spot_sensor_config_2</name>
+  <sdf version="1.7">model.sdf</sdf>
+  <description>
+    Boston Dynamics Spot of CTU-CRAS-Norlab team with 12 breadcrumbs.
+  </description>
+  <depend>
+    <model>
+      <uri>https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/CTU_CRAS_NORLAB_SPOT_SENSOR_CONFIG_1</uri>
```

Comment:
SENSOR_CONFIG_2

Reply:
We don't really have a precedent for this case as normally sensor config 2 does not have its own `model.sdf` file however the _package_ certainly depends on sensor config 1 being present as its `vehicle_topics.launch` is leveraged.

Another reason there doesn't appear to be a precedent for this is that `model.config` files seem to be exceedingly rare in breadcrumb sensor configs.

Reply:
There is a precedent now :) https://github.com/osrf/subt/blob/master/submitted_models/ctu_cras_norlab_marv_sensor_config_2/model.config